### ISSUE TITLE
`stretchy="true"` is probably unnecessary to specify for the fences

### ISSUE THREAD

**Issue by tmke8** [tmke8/latex2mmlc#252]
At least for the common bracket types, this is set automatically. But I need to check that it works for *all* fence symbols.

**Comment by tmke8** [comment]
It seems to be needed for this, at least on Firefox:

```html
<mrow>
  <mo stretchy="true">∕</mo>
  <mfrac>
    <mn>1</mn>
    <mn>2</mn>
  </mfrac>
  <mo stretchy="true">∖</mo>
</mrow>
```

**Comment by tmke8** [comment]
See also
- #260

**Comment by tmke8** [comment]
Was probably solved by #270
